README.md

```diff
@@ -1,25 +1,50 @@
-# CSHub [![Build Status](https://travis-ci.com/finitum/CSHub.svg?branch=dev)](https://travis-ci.com/finitum/CSHub)
-This is a project made for the TU Delft CSE studies, where everyone can post information, summaries and more, for everyone to see and read.
-## Running the project
-* Install [NodeJS](https://nodejs.org/en/), and install the dependencies of the server and client projects (`npm install` in the correct directories)
-* For the server, run `ts-node`, or use the provided IntelliJ runconfigs
-* For the client, run `yarn run serve` :)
+# CSHub
 
-## Feature Requests / Bug Reports
+CSHub started as project made for the TU Delft CSE study to post information, summaries and more, public for everyone.
 
-Create a GitHub issue on this repository, and we will try to take a look at it.
+## Status
 
-## How does it work?
-This project uses TypeScript. This is typed JavaScript, so we can use many advantages of typedness, and use object oriented programming constructs like classes.
+[![Build Status](https://travis-ci.com/finitum/CSHub.svg?branch=dev)](https://travis-ci.com/finitum/CSHub)
+
+## Naming
+
+CSHub was the **C**omputer **S**cience Hub in the first place, but because other studies can use it as well, it is now the **C**ourse **S**ummary Hub.
+
+## Structuring
 
 The project is divided into different sections: 
 
 * Client
 * Server
 * Shared
 
+## Requirements
+
+For this project to run, you need to have [NodeJS](https://nodejs.org/en/) installed.
+
+## Installation
+
+To install both client and server, run `npm install` in the correct directory.
+
+## Run
+
+For the sever:
+`ts-node`
+
+(Or use provided IntelliJ runconfigs.)
+
+For the client:
+`yarn run serve`
+
+## Feature Requests / Bug Reports
+
+Create a GitHub issue on this repository, and we will try to take a look at it.
+
+## How does it work?
```

Comment:
This explanation doesn't really say anything and can be removed all together

Reply:
This test was already in the old README file, so I kept it. Now I removed it in commit 64b42f1862afa021427727893abcbfd4b02b40e6.